extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -145,6 +145,7 @@ export const AnalyticsTrigger = {
   AD_RENDER_START: 'ad-render-start',
   AD_RENDER_END: 'ad-render-end',
   AD_IFRAME_LOADED: 'ad-iframe-loaded',
+  AD_REFRESH: 'ad-refresh',
```

Comment:
Make a comment that this is not part of normal lifecycle triggers.

Reply:
Done.